### ISSUE TITLE
Error while installing playbook

### ISSUE THREAD

**Issue by aminbajand** [aker-gateway/aker-freeipa-playbook#2]
Hi, Thanks about your incredible project.

I installed Ansible. i have 2 machines. one for run Aker and other for remote.

I want to run Aker and all dependencies in one server.
these my servers:
```

192.168.1.20 ==> aker server
192.168.1.21 ==> remote server

```

so i wrote this in "hosts" file:
```
[aker_servers] # Aker server
192.168.1.20


[db_servers]
192.168.1.21

```
then i ran this:
`sudo ansible-playbook -e "install_freeipa=False aker_idp=Json" aker-freeipa.yml`

result:

```
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/ufw.yml' not found,
however since this include is not explicitly marked as 'static: yes', we will try and include it
dynamically later. In the future, this will be an error unless 'static: no' is used on the include
task. If you do not want missing includes to be considered dynamic, use 'static: yes' on the
include or set the global ansible.cfg options to make all inclues static for tasks and/or handlers.

This feature will be removed in a future release. Deprecation warnings can be disabled by setting
deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/elasticsearch-ubuntu.yml'
 not found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/elasticsearch-redhat.yml'
 not found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/logstash-ubuntu.yml' not
found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/logstash-redhat.yml' not
found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/nginx-ubuntu.yml' not
found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/nginx-redhat.yml' not
found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/kibana-ubuntu.yml' not
found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/kibana-redhat.yml' not
found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/filebeat-ubuntu.yml' not
found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/filebeat-redhat.yml' not
found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/json_sample_env.yml' not
found, however since this include is not explicitly marked as 'static: yes', we will try and
include it dynamically later. In the future, this will be an error unless 'static: no' is used on
the include task. If you do not want missing includes to be considered dynamic, use 'static: yes'
on the include or set the global ansible.cfg options to make all inclues static for tasks and/or
handlers.
This feature will be removed in a future release. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.

PLAY [all] *****************************************************************************************

TASK [Gathering Facts] *****************************************************************************
fatal: [192.168.1.20]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: Permission denied (publickey,password).\r\n", "unreachable": true}
fatal: [192.168.1.21]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: Permission denied (publickey,password).\r\n", "unreachable": true}

PLAY RECAP *****************************************************************************************
192.168.1.20               : ok=0    changed=0    unreachable=1    failed=0
192.168.1.21               : ok=0    changed=0    unreachable=1    failed=0


```

I have no experience in ansible. could you help me?

**Comment by anazmy** [comment]
It looks like you didn't download the whole playbook files , I see lots of missing files warnings : 
~~~
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/ufw.yml' not found,
~~~
and
~~~
[DEPRECATION WARNING]: Included file '/home/aminbag/aker-freeipa-playbook/json_sample_env.yml' not
~~~

can you please clone the whole git repo[1] and retry .


[1] https://github.com/aker-gateway/aker-freeipa-playbook

**Comment by leochan007** [comment]
@anazmy json_sample_env.yml seems not existing?

**Comment by ghost** [comment]
json_sample_env.yml seems not existing, same for me.

**Comment by anazmy** [comment]
Can you paste exact error here please

**Comment by anazmy** [comment]
I think I see where the culprit is, will try to address it later when I get access to a laptop/pc

**Comment by anazmy** [comment]
Fixed now in commit 90e300d7ff36c3b3d98785b7224f1328528962ce

Update your local master branch please.
